sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py

```diff
@@ -1083,8 +1075,8 @@ class IssuerProperties(object):
 
     def __init__(self, provider=None, **kwargs):
         # type: (Optional[str], **Any) -> None
-        self._id = kwargs.get("issuer_id", None)
-        self._vault_id = parse_vault_id(self._id)
+        self._id = kwargs.pop("issuer_id", None)
+        self._vault_id = parse_vault_id(self._id) if self._id else None
```

Comment:
Okay, but when `self._vault_id is None`, the `name` and `vault_url` properties will raise. Should a user ever construct this class?

Reply:
got rid of this check, thanks!